tests/core/evm/vm/test_message.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import evm.vm.message as m
+
+message_object = m.Message(
+    1,                      #gas
+    1,                      #gas_price
+    b'',                    #to
+    '11111111111111111111', #sender
+    1,                      #value
+    '',                     #data
+    'asdf',                 #code
+    '22222222222222222222'  #origin
+    )                    
+
+def test_initilizes_properly():
+    # all __init__ parameters are properly validated.
```

Comment:
This would really be testing that all parameters are properly *assigned*.  The code style of this repository typically uses docstrings for this type of comment in the following format.

```python
def test_something():
    """
    Ensure that all initialization parameters are correctly assigned
    """
    ...
```